metrics/tarantool/memory.lua

```diff
@@ -15,6 +16,50 @@ local function update_memory_metrics()
                 nil, nil, {default = true})
         end
     end
+
+    local memory_stat = 0
+
+    if jit.os == 'Linux' then
+        local data = psutils.get_process_stats()
+        memory_stat = data.rss
```

Comment:
It’s confusing that this is the number of pages, not the actual memory size.

On macOS the metric will report bytes, while on Linux it reports pages.

If this goes into a dashboard, it could be misleading.

Maybe we should make two metrics: one for memory calculated through Tarantool calls, which will work on both macOS and Linux; and another one for RSS, which will work only on Linux.